post.hbs

```diff
@@ -66,6 +66,7 @@
                                     {{img_url feature_image size="xxl"}} 4000w
                                 "
                                 sizes="100vw"
```

Comment:
Drobnůstka: různé pořadí atributů u `<img />` napříč šablonami, ale to je detail, kterého jsem si teď všimla.

Reply:
Opraveno